packages/application-components/src/theming.ts

```diff
@@ -0,0 +1,25 @@
+import kebabCase from 'lodash/kebabCase';
```

Comment:
It would probably be better to have an independent package in `app-kit` to have the theming part.